### PR TITLE
fix: clarify shopify provider instructions

### PR DESCRIPTION
**Pull request by paul-at-atama**

### CODE REVIEW

File: docs/reference/providers/shopify.md [hunk 1]
```diff
@@ -6,19 +6,140 @@ To connect to your Shopify account, you'll need the following information:
...
 
 ### Action
 * Get Products
 * Get Product
 * Create cart
 * Add cart items
 * Get Cart
+
+#### Product query
+This is fragment we use for "Get Product(s)" queries:
```
Comment (s-atama) [inline docs/reference/providers/shopify.md]:
```suggestion
This is the fragment we use for "Get Product(s)" queries:
```

File: docs/reference/providers/shopify.md [hunk 2]
```diff
@@ -6,19 +6,140 @@ To connect to your Shopify account, you'll need the following information:
...
+          }
+        }
+        cursor
+      }
+    }
+  }
+```
+
+#### Cart query
+This is fragment we use for the "Get Cart" query:
```
Comment (s-atama) [inline docs/reference/providers/shopify.md]:
```suggestion
This is the fragment we use for the "Get Cart" query:
```

**Review by s-atama — APPROVED** [review]

File: docs/reference/providers/shopify.md [hunk 3]
```diff
@@ -6,19 +6,140 @@ To connect to your Shopify account, you'll need the following information:
...
+* unauthenticated_read_content
+* unauthenticated_read_metaobjects
+* unauthenticated_read_product_listings
+* unauthenticated_read_product_tags
+
 ## Supported capabilities
 
 ### Data
-* Products
+* Products – You have access to everything from the [Product API](https://shopify.dev/docs/api/admin-graphql/2023-01/objects/Product), even if it's not shown in the JSON schema or graphs. This may happen because we currently don't follow connections.
```
Comment (ryatama) [inline docs/reference/providers/shopify.md]: have you tested that this works? on the data side we aren't doing a 'proxy' where they pass through a query, we're just using our metadata field `queryBody` that contains the actual query we're going to run to get the product

Comment (paul-at-atama) [reply docs/reference/providers/shopify.md]: ah, you're right, this is GraphQL so it works differently and won't return fields unless specifically mentioned. I'll clarify the statement.

**Pull request merged**